components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -142,8 +144,17 @@ const ContributorProfileHeader = ({
             size={184}
             isCircle
           />
+
+          <Link
+            href={cardPageUrl(username!)}
+            className="absolute bottom-0 z-10 grid w-12 h-12 rounded-full shadow-md place-content-center border-conic-gradient right-4"
+          >
+            <div className="grid w-10 h-10 overflow-hidden rounded-full place-content-center bg-black/80">
+              <Image priority alt="user profile cover image" className="w-5 h-6 " src={PizzaGradient} />
```

Comment:
Can we change the width/height classes to "w-6 h-7" ? I want to make the icon a bit bigger